stdlib/Unicode/src/Unicode.jl

```diff
@@ -4,19 +4,12 @@ __precompile__(true)
 
 module Unicode
 
-using Base.Unicode: normalize, graphemes, isassigned, textwidth, isvalid,
-                    islower, isupper, isalpha, isdigit, isxdigit, isnumeric, isalnum,
-                    iscntrl, ispunct, isspace, isprint, isgraph,
-                    lowercase, uppercase, titlecase, lcfirst, ucfirst, iscased
+using Base.Unicode: normalize, graphemes, isassigned, iscased
```

Comment:
Why have you kept `iscased` in Unicode? It feels lonely now that all case functions have been moved back to Base.